src/_common/game/thumbnail-img/thumbnail-img.ts

```diff
@@ -39,6 +40,23 @@ export default class AppGameThumbnailImg extends Vue {
 		return this.hasVideo && ContentFocus.hasFocus;
 	}
 
+	get videoSources(): VideoSourceArray {
```

Comment:
needs to handle the case where mp4 url or webm url aren't set. they might be empty/null and we shouldn't return it as part of the array then